scripts/Engineering/gui/engineering_diffraction/tabs/fitting/plotting/plot_view.py

```diff
@@ -45,7 +45,13 @@ def setup_figure(self):
         self.toolbar = FittingPlotToolbar(self.figure.canvas, self, False)
         self.vLayout_plot.addWidget(self.toolbar)
         self.vLayout_plot.addWidget(self.figure.canvas)
-        self.fit_browser = EngDiffFitPropertyBrowser(self.figure.canvas, ToolbarStateManager(self.toolbar)) # self.fitprop_toolbar
+        self.fit_browser = EngDiffFitPropertyBrowser(self.figure.canvas,
+                                                     ToolbarStateManager(self.toolbar))
+        hide_props = ['StartX', 'EndX', 'Minimizer', 'Cost function', 'Max Iterations', 'Output',
```

Comment:
Might be worth declaring this array toward the top of the file to allow easier future changes

Reply:
Happy to do this but I think it makes more sense to keep it where all the other fit_browser stuff is (already near the top) as I don't think this will be changed once we have settled on the options they want. Of course I don't need to define a variable at all - would you rather I write out the list in the function argument?